habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml

```diff
@@ -3,23 +3,37 @@ CHECKPOINT_FOLDER: "/checkpoint/eundersander/gala_kinematic/ckpt/gala_kinematic_
 VIDEO_DIR: "../videos2"
 VIDEO_OPTION: ["disk"]
 REWARD_SCALE: 1.0 # 0.01
-NUM_CHECKPOINTS: 10
+NUM_CHECKPOINTS: 100
 BATCHED_ENV: True
 OVERLAP_PHYSICS: True
-SAVE_VIDEOS_INTERVAL: 500
+SAVE_VIDEOS_INTERVAL: -1
 LOG_INTERVAL: 10
 NUM_UPDATES: -1
 TOTAL_NUM_STEPS: 5e9
-NUM_ENVIRONMENTS: 512
+NUM_ENVIRONMENTS: 128
 MAX_EPISODE_LENGTH: 500
+NUM_EPISODES: 10000
+MIN_NON_TARGET: 27
+MAX_NON_TARGET: 32
+FIXED_ROBOT_START_POSITION: False
+PREVENT_STOP_ACTION: False
 STAGGER: True
+TEST_EPISODE_COUNT: 10
+EVALUATION_MODE: False
+TRAIN_DATASET: ../data/train_generated.episode_set.json
```

Comment:
Will you add these data files to `gala_kinematic master`?

Reply:
I always override these when running code. We should get Andrew's dataset on master, then I will like the tidy house directly.